docs/src/modules/components/MarkdownDocs.js

```diff
@@ -40,56 +41,64 @@ function MarkdownDocs(props) {
   return (
     <MarkdownDocsContents markdown={markdown} markdownLocation={markdownLocationProp}>
       {({ contents, markdownLocation }) => (
-        <AppFrame>
-          <Head
-            title={`${headers.title || getTitle(markdown)} - Material-UI`}
-            description={headers.description || getDescription(markdown)}
-          />
-          <AppTableOfContents contents={contents} />
-          {disableAd ? null : (
-            <Portal container={() => document.querySelector('.description')}>
-              <Ad />
-            </Portal>
-          )}
-          <AppContent className={classes.root}>
-            <div className={classes.header}>
-              <EditPage
-                markdownLocation={markdownLocation}
-                sourceCodeRootUrl={SOURCE_CODE_ROOT_URL}
+        <PageContext.Consumer>
+          {({ userLanguage }) => (
+            <AppFrame {...{ userLanguage }}>
```

Comment:
It's simpler to understand for our users:
```jsx
<AppFrame userLanguage={userLanguage}>
```

Reply:
Can't see any but advanced users need to go near this, but fine.